SECURITY.md

```diff
@@ -1,34 +1,93 @@
 # Security policy
 
-## Supported versions
+## Warning to users
+
+PowSyBl is a library dedicated to electrical grid modelling and simulation. As such, it includes data importers to feed the calculations. Please make sure your data comes from a safe source.
```

Comment:
There is also some visualisation tools to define the scope. 
You have data importers: network data (XML, JSON, txt), parameter data (JSON), entry data for simulations (JSON).
I think that "make sure your data comes from a safe source" is not enough. @jonenst we cannot load any kind of file in Powsybl, for XML we have a strong format to follow (xsd). Are we really able to handle unsafe data? Same for CIM format, PowerFactory format, PSSE format, IEEE format, UCTE format. In the way we have designed our importers, can we load a dangerous command line?

Reply:
After discussing this point with @jonenst and @geofjamg: we apply Sonar rules and we thoroughly review Pull Requests to avoid security flaws. Anyway, it is an added security to encourage users to check their data. I will rephrase it in a new version. We will see if it suits everyone.